lucene/core/src/java/org/apache/lucene/search/MultiCollector.java

```diff
@@ -115,6 +116,11 @@ public ScoreMode scoreMode() {
     return scoreMode;
   }
 
+  // nocommit: need to raise a LUCENE jira for this?
+  public List<Collector> getCollectors() {
```

Comment:
This change is small and simple enough that I suggest just at-mentioning one or two people you think would care about this to get their opinion if it's worth a separate JIRA, and separately if it's even worth a Lucene changes.txt entry.  I think not worth it.  Remember the higher javadoc standards :-)